Content.Server/_NF/Shuttles/Systems/ShuttleSystem.cs

```diff
@@ -29,20 +36,21 @@ private void OnToggleStabilizer(EntityUid uid, ShuttleConsoleComponent component
         {
             InertiaDampeningMode.Off => 0,
             InertiaDampeningMode.Dampen => shuttleComponent.LinearDamping,
-            InertiaDampeningMode.Anchored => 1,
-            _ => 999999,// if it's not the above it's a station, they shouldn't be able to change the dampening, should we alert the admin?
+            InertiaDampeningMode.Anchored => AnchorDampeningStrength,
+            _ => StationaryAnchorStrength,// if it's not the above it's a station, they shouldn't be able to change the dampening, should we alert the admin?
```

Comment:
Shuttles should not be able to set their anchor strength to make them immobile.

Reply:
they cant.  if the status is not set to one of the 3 above, then it is a hacked client.  and logs will show us that this has happened